bcrhp/src/bcrhp/pages/steps/Step5_RecognitionDetails.vue

```diff
@@ -130,7 +150,8 @@ onMounted(() => {
                 <DatePicker
                     id="designationDate"
                     ref="designationDateField"
-                    v-model="heritageSite.designationDate"
+                    v-model="designationDate"
```

Comment:
Should we be using `heritageSite.recognitionDetails.designationDate` here to match the pattern used elsewhere?